example/llm/flake.nix

```diff
@@ -43,7 +54,8 @@
                 RAG_EMBEDDING_ENGINE = "ollama";
                 RAG_EMBEDDING_MODEL = "mxbai-embed-large:latest";
                 RAG_EMBEDDING_MODEL_AUTO_UPDATE = "True";
-                RAG_RERANKING_MODEL_AUTO_UPDATE = "True";              };
```

Comment:
My VSCode autoformatted the braces here. Somehow our nix flake check for formatting is not working. @shivaraj-bh

Reply:
https://github.com/juspay/services-flake/issues/238